### ISSUE TITLE
Address PytzUsageWarning

### ISSUE THREAD

**Issue by lfarvour** [demisto/demisto-sdk#2035]
Recently while using `demisto-client` with a fresh installation of `python 3.8.6`, I started receiving the following warnings:

```
[USER_DIR]/.pyenv/versions/3.8.6/envs/clean-test/lib/python3.8/site-packages/demisto_client/__init__.py:199: PytzUsageWarning: The localize method is no longer necessary, as this time zone supports the fold attribute (PEP 495). For more details on migrating to a PEP 495-compliant implementation, see https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html
  value = tzlocal.get_localzone().localize(value)
```

**Comment by lfarvour** [comment]
This is still an issue.

**Comment by dorschw** [comment]
Thanks for opening the issue, @lfarvour 

We've opened an internal issue to solve it, please note this is a low priority as the problem is mostly cosmetic, and does not affect usage of the SDK, and so may take a while until handled. 

We'll close this issue once a fix is merged.

**Comment by lfarvour** [comment]
Great, thanks! 

I know it's non-breaking, but it's noisy :) Thanks again; I appreciate you taking it internal and prioritizing appropriately. Hopefully it's a quick fix.

**Comment by lfarvour** [comment]
Oi, accidental close... Don't click buttons late in the day on a Monday...
